draft-ietf-add-split-horizon-authority.xml

```diff
@@ -329,32 +331,33 @@
 
     <section>
       <name>Examples of Split-Horizon DNS Configuration</name>
-      <t>Two examples are shown below. The first example showing an company
-      with an internal-only DNS server resolving the entire zone for that
-      company (e.g., *.example.com) the second example resolving only a
-      subdomain of the company's zone (e.g., *.internal.example.com).</t>
+      <t>Two examples are shown below. The first example shows a company
+      with an internal-only DNS server that claims the entire zone for that
+      company (e.g., <tt>*.example.com</tt>). In the second example, the
+      internal servers resolves only a
+      subdomain of the company's zone (e.g., <tt>*.internal.example.com</tt>).</t>
       <section anchor="internal-only">
         <name>Split-Horizon Entire Zone</name>
         <t>Consider an organization that operates "example.com", and runs a
         different version of its global domain on its internal network. Today,
         on the Internet it publishes two NS records, "ns1.example.com" and
         "ns2.example.com".</t>
-        <t>The host and network first need mutual support one of the
-        mechanisms described in <xref target="learning"/>. Shown
-        in <xref target="fig-learn"/> is learning using DNR and PvD.</t>
+        <t>First, the host and network both need to support one of the discovery
+        mechanisms described in <xref target="learning"/>. <xref target="fig-learn"/>
+        shows discovery using DNR and PvD.</t>
         <t>Validation is then perfomed using either <xref
-        target="example-verify-public">Public DNS</xref> or <xref
+        target="example-verify-external">an external resolver</xref> or <xref
         target="example-verify-dnssec">DNSSEC</xref>.</t>
         <ul empty="true">
-          <li><strong>steps 1-2</strong>: The client determines the network's DNS
+          <li><strong>Steps 1-2</strong>: The client determines the network's DNS
             server (ns1.example.com) and Provisioning Domain (pvd.example.com)
             using <xref target="I-D.ietf-add-dnr">DNR</xref> and <xref
             target="RFC8801">PvD</xref>, using one of DNR Router Solicitation,
             DHCPv4, or DHCPv6.</li>
-          <li><strong>step 3-5</strong>: The client connects to the DNR-learned DNS
-            server (ns1.example.com), validates its certificate, and queries
-            for pvd.example.com.</li>
-          <li><t><strong>steps 6-7</strong>: The client connects to the PvD server,
+          <li><strong>Step 3-5</strong>: The client connects to the DNS
+            server learned from DNR (ns1.example.com), validates its certificate, and
```

Comment:
Replace "validates its certificate" with "it will authenticate the server to its name using TLS (Section 8 of RFC8310])" or modify this step to align with the step in other examples.